pkg/operator/ceph/pool/radosnamespace/controller.go

```diff
@@ -233,10 +258,16 @@ func (r *ReconcileCephBlockPoolRadosNamespace) reconcile(request reconcile.Reque
 			// We must remove it first otherwise the checker will panic since the status/info will be nil
 			r.cancelMirrorMonitoring(radosNamespaceChannelKeyName(radosNamespace.Namespace, poolAndRadosNamespaceName))
 		}
-		err = csi.SaveClusterConfig(r.context.Clientset, buildClusterID(radosNamespace), cephCluster.Namespace, r.clusterInfo, nil)
-		if err != nil {
-			return reconcile.Result{}, radosNamespace, errors.Wrap(err, "failed to save cluster config")
+
+		if len(cephRNSList.Items) <= 1 {
```

Comment:
This is the same check as above. Can you just move `SaveClusterConfig()` inside the same conditional block above (after line 259)?

Reply:
Discussed, and will keep this separate check to allow the SaveClusterConfig() call for external clusters, even though not sure it's necessary.